core/src/test/java/com/taobao/arthas/core/util/StringUtilsTest.java

```diff
@@ -18,4 +27,388 @@ public void testHumanReadableByteCount() {
+        Assert.assertEquals("11", StringUtils.join(new Object[] {11}, null));
+        Assert.assertEquals("", StringUtils.join(new Object[] {}, null));
+        Assert.assertEquals("", StringUtils.join(new Object[] {}, "HH"));
+        Assert.assertEquals("foobarbaz", StringUtils.join(new Object[] {"foo", "bar", "baz"}, ""));
+    }
+
+    @Test
+    public void testLength() {
+        Assert.assertEquals(0, StringUtils.length(null));
+        Assert.assertEquals(0, StringUtils.length(""));
+        Assert.assertEquals(3, StringUtils.length("boo"));
+    }
+
+    @Test
+    public void testNormalizeClassName() {
+        Assert.assertEquals(".", StringUtils.normalizeClassName("/"));
+        Assert.assertEquals(",.o\'+$g--.", StringUtils.normalizeClassName(",.o\'+$g--."));
+        Assert.assertEquals("", StringUtils.normalizeClassName(""));
+    }
+
+    @Test
+    public void testObjectToString() {
+        Assert.assertEquals("1", StringUtils.objectToString(1));
+        Assert.assertEquals("", StringUtils.objectToString(null));
+        Assert.assertEquals("{}", StringUtils.objectToString(new TreeMap()));
+    }
+
+    @Test
+    public void testQuoteIfString() {
+        Assert.assertEquals("\'foo\'", StringUtils.quoteIfString("foo"));
+        Assert.assertEquals(1, StringUtils.quoteIfString(1));
+    }
+
+    @Test
+    public void testQuote() {
+        Assert.assertEquals("\'!\'", StringUtils.quote("!"));
+        Assert.assertEquals("\'1234\'", StringUtils.quote("1234"));
+    }
+
+    @Test
+    public void testRepeat() {
+        Assert.assertNull(StringUtils.repeat(null, -1635778558));
+        Assert.assertEquals("", StringUtils.repeat("", 3));
+        Assert.assertEquals("\u0000\u0000", StringUtils.repeat("\u0000", 2));
+        Assert.assertEquals("\u0000\u0000\u0000\u0000", StringUtils.repeat("\u0000\u0000", 2));
+        Assert.assertEquals("", StringUtils.repeat("foo", 0));
+        Assert.assertEquals("foo", StringUtils.repeat("foo", 1));
+        Assert.assertEquals("foofoofoofoo", StringUtils.repeat("foo", 4));
+
+        StringBuilder sb = new StringBuilder();
+        for (int i = 0; i < 8194; i++)
+            sb.append('c');
+        Assert.assertEquals(sb.toString(), StringUtils.repeat('c', 8194));
+
+        thrown.expect(NegativeArraySizeException.class);
+        StringUtils.repeat("\uffff\u2aa0", 1073742337);
+    }
+
+    @Test
+    public void testReplace() {
+        Assert.assertEquals(" ", StringUtils.replace(" ", "", "    "));
+        Assert.assertEquals("", StringUtils.replace("", "", "    "));
+        Assert.assertEquals("baz", StringUtils.replace("bar", "r", "z"));
+    }
+
+    @Test
+    public void testSplitArrayElementsIntoProperties() {
+        final String[] array = {};
+        Assert.assertNull(StringUtils.splitArrayElementsIntoProperties(array, "_", "DEE"));
+
+        final String[] array2 = {"1=foo", "2=bar"};
```

Comment:
`foo=1` would make more intuitive sense to me as properties are `key=value` things.